leshan-server-core/src/main/java/org/eclipse/leshan/server/queue/Presence.java

```diff
@@ -20,7 +20,7 @@
  */
 public enum Presence {
     /** LWM2M Client is reachable and messages can be sent **/
-    ONLINE,
+    AWAKE,
     /** LWM2M Client is not reachable and no messages can be sent **/
-    OFFLINE
+    SLEEPING

```

Comment:
Naming: I would prefer ONLINE/OFFLINE but as the spec use both words, I think both are acceptable.